components/Controllers/AuctionPageContent/AuctionPageContent.tsx

```diff
@@ -183,7 +181,7 @@ export function AuctionPageContent({
               auctionUnderlyingPrice={liveAuctionPriceUnderlying}
               priceUpdated={priceUpdated}
               timeElapsed={timeElapsed}
-              topBid={topBid}
+              topBid={topBid!}
```

Comment:
annoying the typechecker isn't smart enough to recognize bools in memos

Reply:
You might be able to use a [type predicate](https://www.typescriptlang.org/docs/handbook/advanced-types.html#using-type-predicates), like
```typescript
  const topBid = useAuctionTopBid(auction);
  const auctionPageLoading = useMemo((): topBid is number => {
    return !topBid || !paprPrice;
  }, [topBid, paprPrice]);
```
not 100% sure whether it will work with the useMemo on there, could always not memoize since this is not a computationally intensive function

Reply:
I think type guards can only work on a single thing though